docs/docs/en/guide/howto/general-setting.md

```diff
@@ -0,0 +1,22 @@
+# General Setting
+
+## Language
+
+DolphinScheduler supports two types of built-in language which include `English` and `Chinese`. You could click the button
+on the top control bar named `English` and `Chinese` and change it to another one when you want to switch the language. All
+your pages of DolphinScheduler's language will be changed when you switch the language from one to another.
```

Comment:
All
your pages of DolphinScheduler's language will be changed when you switch the language from one to another.
->
The entire DolphinScheduler page language will shift when you switch the language selection.

Reply:
Fix